automation/run-tests.sh

```diff
@@ -64,17 +67,31 @@ function install_nmstate {
 }
 
 function run_tests {
-    docker_exec "
-      cd /workspace/nmstate &&
-      pytest \
-        --verbose --verbose \
-        --log-level=DEBUG \
-        --durations=5 \
-        --cov=\$($PYTHON_SITE_PATH_CMD)/libnmstate \
-        --cov=\$($PYTHON_SITE_PATH_CMD)/nmstatectl \
-        --cov-report=html:htmlcov-integ \
-        tests/integration \
-    ${nmstate_pytest_extra_args}"
+    if [ $TEST_TYPE == "all" ] || [ $TEST_TYPE == "unit_code_style" ];then
+        docker_exec 'cd /workspace/nmstate && tox -e flake8,pylint'
```

Comment:
We could just build the command in the "if/s" and execute it once after.

`docker_exec 'cd /workspace/nmstate && ` is duplicated all over.

Just a nice to have, not a must.

Reply:
Done.